api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -314,6 +316,11 @@ message Watchdog {
   // duration assume a true deadlock and kill the entire Envoy process. Set to 0
   // to disable this behavior. If not specified the default is 0 (disabled).
   google.protobuf.Duration multikill_timeout = 4;
+
+  // Sets threshold for multikill timeout in terms of the percentage of
+  // Watchdogs being nonresponsive for at least the multikill_timeout.
```

Comment:
nit: watchdogs -> threads

 percentage of nonresponsive threads required for the multikill_timeout.